highlight.io/components/QuickstartContent/logging/java/other.tsx

```diff
@@ -0,0 +1,49 @@
+import { siteUrl } from '../../../../utils/urls'
+import { QuickStartContent } from '../../QuickstartContent'
+import { previousInstallSnippet, verifyLogs } from '../shared-snippets'
+
+export const JavaOtherLogContent: QuickStartContent = {
+	title: 'Java',
+	subtitle: 'Learn how to set up highlight.io Java log ingestion.',
+	logoUrl: siteUrl('/images/quickstart/java.svg'),
+	entries: [
+		previousInstallSnippet('java'),
+		{
+			title: 'Install the Highlight Java SDK.',
+			content: 'Add Highlight to your maven pom file.',
+			code: {
+				text: `<dependency>
+	<groupId>io.highlight</groupId>
+	<artifactId>highlight-sdk</artifactId>
+	<version>latest</version>
+</dependency>`,
+				language: 'xml',
+			},
+		},
+		{
```

Comment:
do we want the `Highlight.builder` step here too with the project id?

Reply:
Yes, that's a great idea